<table>
<tr>
        <td font-weight:bold>Functions</td>
        <td font-weight:bold>Applications</td>
        <td font-weight:bold>Input files</td>
        <td font-weight:bold>Main output files</td>
        <td font-weight:bold>Programs</td>
        <td font-weight:bold>References</td>
</tr>
<tr>
        <td>Characterize Nucleic-acid Feature</td>
        <td>Explore the similarity between assembled and annotated transcripts in terms of the distribution of transcript length, exon length, G+C content</td>
        <td>Confident transcripts; Reference transcripts</td>
        <td>Diagnostic plots (barplot/density)</td>
        <td>In-house scripts</td>
        <td>Our study</td>
</tr>
<tr>
        <td>Characterize Amino-acid Feature</td>
        <td>Explore the similarity between assembled and annotated transcripts in terms of the distribution of amino acid-based features used in SAT</td>
        <td>Confident transcripts; Reference transcripts</td>
        <td>Diagnostic plots (barplot/density)</td>
        <td>In-house scripts</td>
        <td>Our study</td>
</tr>
<tr>
        <td>Detect alternative splicing events</td>
        <td>Detect alternative splicing events in assembled transcripts</td>
        <td>Alignment results (BAM); All transcript annotation file (GTF)</td>
        <td>Alternative splicing events; Number of support reads</td>
        <td>SGSeq (version 3.8; https://www.bioconductor.org/packages/release/bioc/html/SGSeq.html)</td>
        <td>Goldstein <I>et al</I>., 2016</td>
</tr>
</table>
